languages/fr.json

```diff
@@ -0,0 +1,41 @@
+{
+	"global": {
+		"toggle-theme": "Changer le thème",
+		"toggle-lang": "Langue"
+	},
+	"nav": {
+		"gallery": "Galerie",
+		"meteoriks": "Meteoriks",
+		"about": "À propos"
+	},
+	"artwork": {
+		"platform": "{{platform}} executable"
+	},
+	"badges": {
+		"meteorik-nominee": "Nominé aux Meteoriks {{meteorik-year}}",
+		"meteorik-winner": "Gagnant des Meteoriks {{meteorik-year}}"
```

Comment:
"Gagnant" here should probably be "Lauréat•e" in this context, both for translation correctness (Meteoriks are an award type of thing rather than just pure competition) and for inclusivity. 🙂

Reply:
Ha oui ! bien trouvé